README.md

```diff
@@ -1,8 +1,14 @@
-# Example Plugin for GLPI
+# GLPi Example Plugin
 
 ## Introduction
 
-This plugin aims to introduce how to interact with glpi HOOKS.  
-Most of implemented features are for example and do nothing.  
- 
-See [documentation](https://github.com/pluginsGLPI/example/wiki) for plugin creation
+This plugin implements a few basic glpi hooks.
+Most of implemented features stands for example and are not
+expected to provide fantastic functionality.
+
```

Comment:
Personal touch, I tried not to change the meaning, PR still can be rejected.
I think the developer implements available hooks in a way, I wanted to put an emphasis on it.